core/chains/evm/client/errors.go

```diff
@@ -102,11 +102,12 @@ var besu = ClientErrors{
 
 // Arbitrum
 // https://github.com/OffchainLabs/arbitrum/blob/cac30586bc10ecc1ae73e93de517c90984677fdb/packages/arb-evm/evm/result.go#L158
-var arbitrumFatal = regexp.MustCompile(`(: |^)(invalid message format|forbidden sender address|execution reverted: error code)$`)
+// nitro:
```

Comment:
- [ ] Link to nitro source

Should we consider a separate grouping for Arbitrum _Nitro_ errors, just for organization?